data/json/vehicle_parts.json

```diff
@@ -1248,18 +1237,15 @@
     },{
         "type" : "vehicle_part",
         "id" : "reinforced_solar_panel_v2",
-        "name" : "upgraded reinforced solar panel",
-        "symbol" : "#",
+        "copy-from": "solar_panel_v2",        
```

Comment:
Why is there all these spaces?

Reply:
F*ck me... no clue what vscode (or more probably me) did there...